test/rxd/ecs/test_ecs_include_flux.py

```diff
@@ -37,7 +44,8 @@ def test_ecs_include_flux(ecs_include_flux):
 
 
 def test_ecs_include_flux_cvode(ecs_include_flux):
-    return
```

Comment:
Hello @adamjhn 
I reverted the `return` removal as the test produces a segment fault at least on my side and @iomaganaris . I do not know if this test is supposed to pass.
cc @ohm314

Reply:
Here is a gdb stacktrace of the SEGV:

```
Core was generated by `/usr/bin/python3 -m pytest /home/tristan/src/github.com/neuronsimulator/nrn/tes'.
Program terminated with signal SIGSEGV, Segmentation fault.
#0  0x00007f1f2ad0921d in ecs_do_reactions (dataptr=0x7f1f3d3a4420)
    at /home/tristan/src/github.com/neuronsimulator/nrn/src/nrnpython/rxd_extracellular.cpp:434
434	                                        v_set_val(x, j, v_get_val(x, j) - m_get_val(jacobian, j, k) * v_get_val(x, k));
(gdb) bt
#0  0x00007f1f2ad0921d in ecs_do_reactions (dataptr=0x7f1f3d3a4420)
    at /home/tristan/src/github.com/neuronsimulator/nrn/src/nrnpython/rxd_extracellular.cpp:434
#1  0x00007f1f2ad095c8 in ecs_do_reactions (dataptr=0x0)
    at /home/tristan/src/github.com/neuronsimulator/nrn/src/nrnpython/rxd_extracellular.cpp:500
#2  0x00007f1f2ad0a6d5 in do_currents (grid=0x27fa0a0, output=0x2858868, dt=2.0865494978397359e-316, grid_id=0)
    at /home/tristan/src/github.com/neuronsimulator/nrn/src/nrnpython/rxd_extracellular.cpp:706
#3  0x00007f1f2abc9b0c in print_syn () at /home/tristan/src/github.com/neuronsimulator/nrn/src/nrnoc/synapse.c:72
#4  0x00007f1f2abc9c09 in print_syn () at /home/tristan/src/github.com/neuronsimulator/nrn/src/nrnoc/synapse.c:69
#5  0x00007f1f2ab8d90a in pr_realcell (ps=..., nt=..., f=0x7f1f2abc9c09 <print_syn+315>)
    at /home/tristan/src/github.com/neuronsimulator/nrn/src/nrniv/prcellstate.cpp:161
#6  0x00007f1f2ab0cae0 in KSChan::gate_remove (this=0x21dfa30, i=0)
    at /home/tristan/src/github.com/neuronsimulator/nrn/src/nrniv/kschan.cpp:1807
#7  0x00007f1f2abe44aa in hoc_coredump_on_error ()
    at /home/tristan/src/github.com/neuronsimulator/nrn/src/oc/hoc.c:741
#8  0x00007f1f2ab0c936 in KSChan::gate_remove (this=0x286f610, i=0)
    at /home/tristan/src/github.com/neuronsimulator/nrn/src/nrniv/kschan.cpp:1802
#9  0x00007f1f2accdd53 in IDASetCoeffs (IDA_mem=0x0, ck=0x0)
    at /home/tristan/src/github.com/neuronsimulator/nrn/src/sundials/ida/ida.c:1571
#10 0x00007f1f2accdbf7 in IDAStep (IDA_mem=0x7f1f2accdbf7 <IDAStep+422>)
    at /home/tristan/src/github.com/neuronsimulator/nrn/src/sundials/ida/ida.c:1537
#11 0x00007f1f2acccf64 in IDAStopTest1 (IDA_mem=0x286f610, tout=9.6910731375200172e-315, 
    tret=0x7f1f2acccf64 <IDAStopTest1+1056>, yret=0x7ffe74ea0a00, ypret=0x3ff0000000000000, itask=-1074790400)
    at /home/tristan/src/github.com/neuronsimulator/nrn/src/sundials/ida/ida.c:1276
#12 0x00007f1f2acca9b4 in IDAMalloc (ida_mem=0x0, res=0x65, t0=0, yy0=0x0, yp0=0x81a71d40, itol=0, rtol=0x286f610, 
    atol=0x28b2990) at /home/tristan/src/github.com/neuronsimulator/nrn/src/sundials/ida/ida.c:256
#13 0x00007f1f2acc6c58 in CVSpgmrSetDelt (cvode_mem=0x286f610, delt=99999.999999997774)
    at /home/tristan/src/github.com/neuronsimulator/nrn/src/sundials/cvodes/cvspgmr.c:311
#14 0x00007f1f2ab0c24d in KSChan::state_remove (this=0x21dfa30, i=0)
    at /home/tristan/src/github.com/neuronsimulator/nrn/src/nrniv/kschan.cpp:1750
#15 0x00007f1f2ab0baf2 in KSChan::check_struct (this=0x40f869ffffffff67)
    at /home/tristan/src/github.com/neuronsimulator/nrn/src/nrniv/kschan.cpp:1694
#16 0x00007f1f2ab4e407 in std::__copy_move_backward<false, false, std::random_access_iterator_tag>::__copy_move_b<SelfEventPPTable_Entry*, SelfEventPPTable_Entry*> (__first=0x271b528, __last=0x2417a00, __result=0x21dfa30)
    at /usr/include/c++/9/bits/stl_algobase.h:547
#17 0x430c6bf526340000 in ?? ()
#18 0x0000000002215b20 in ?? ()
#19 0x00000000021dfa30 in ?? ()
#20 0x0000000000000001 in ?? ()
#21 0x0000000002843ef0 in ?? ()
#22 0x0000000100000000 in ?? ()
#23 0xfe30ad047298af00 in ?? ()
#24 0x00007ffe74ea0c50 in ?? ()
#25 0x00007f1f2ab4e0ae in std::_Rb_tree<long, std::pair<long const, Point_process*>, std::_Select1st<std::pair<long const, Point_process*> >, nrnhash_lt_SelfEventPPTable, std::allocator<std::pair<long const, Point_process*> > >::_M_get_insert_unique_pos (this=0x40f869ffffffff67, __k=@0x2417a00: 4614911179122839157)
    at /usr/include/c++/9/bits/stl_tree.h:2089
Backtrace stopped: previous frame inner to this frame (corrupt stack?)
```